docs/components.md

```diff
@@ -158,8 +158,419 @@ You should see this:
 
 ## _Multistep forms_
 
+A multi-step form is a long form that is broken into multiple pieces. 
+They're used to make long forms, such as shipping or registration forms, less intimidating and daunting. 
+By allowing customers to complete their information in smaller chunks, you create a positive user experience and increase conversions.
 ---
 
+
+>**NOTE**
+> To be able to follow these steps you need an overview and detail page. If you haven't already done this, click [here](pages.md) for the explanation
+
+You always have a beginning and an end of a multi-step form. We can make this generic so that we can reuse it. This is what we're going to start with first.
+
+We will create a template at `cd /pwa/src/templates/multiStep/StartTemplate.tsx`.
+In this template we want to divide the structure into three parts with a devider.
+
+1. The subject with a description.
+2. The steps to take with a button to continue.
+3. More information where to refer to another page
+
+We are going to start adding the first part, the subject with a description.
+Because the title and description can change when reusing this template, we will create an interface for this.
+The code looks like this.
+```Javascript
+//  /src/templates/test/TestTemplate.tsx
+import * as React from "react";
+
+interface StartTemplateProps {
+  title: string;
+  description: string;
+
+export const StartTemplate: React.FC<StartTemplateProps> = ({
+  title,
+  description,
+}) => {
+
+  return (null);
+};
+```
+
+Then in the return of the template we add a div that will contain everything.
+In this we add another div where we add a `<Heading>{title}</Heading>` and `<Paragraph>{description}</Paragraph>` of the package `@gemeente-denhaag/components-react`.
+
+```Javascript
+//  /src/templates/test/TestTemplate.tsx
+import * as React from "react";
+
+interface StartTemplateProps {
+  title: string;
+  description: string;
+}
+
+export const StartTemplate: React.FC<StartTemplateProps> = ({
+  title,
+  description,
+}) => {
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.title}>
+        <Heading1>{title}</Heading1>
+        <Paragraph>{description}</Paragraph>
+      </div>
+    </div>
+  );
+};
+```
+
+We close the first section with a `<Divider />`, then we can continue with the second section of the template.
```

Comment:
Try to write not from a we (can't always escape that, e.g. we as a tema made a choice), but preferrably stick to what the code does. Instead of "we'll open another div", "Another div is needed here" or "Adding a div for " .

You on the other hand is often find, since the documentation is made for the reader.